views/questions-list.pug

```diff
@@ -0,0 +1,19 @@
+extends layout 
+
+block content 
+    .body-content
+        - noun = (questions.length === 1) ? noun : noun + 's'
```

Comment:
Lines 5 and 6 make the noun singular if there's only 1, and plural otherwise. Documenting that here since the diff doesn't show it for renamed files